src/pages/ui-update-sample.mdx

```diff
@@ -0,0 +1,122 @@
+
+To set the variable value for a tenant/environment, click the **Edit** (pencil) icon on the right-side of the table row. This will open a dialog where you can update the value, then click **Update** to save the new value.
+
+:::figure
+![](/docs/projects/variables/images/tenant-variables-edit.png "width=500")
+:::
+
+## Common templates
+
+To specify common variables that can be used across multiple tenants, you'll need to add a variable template to either an existing or new [library variable set](/docs/projects/variables/library-variable-sets). You'll also need to [include the variable set in the project](/docs/projects/variables/library-variable-sets#adding-the-library-variable-set-to-a-project) to manage any template values.
+
+To manage common template values, navigate to the **Tenant Variables** page in any project and select the **Common Templates** tab.
+
+The variable management experience for common templates is similar to project templates, however the side list will show included library variable sets and the values will be grouped by variable set.
+
+To edit a variable value, click the **Edit** icon on the right-side of a table row, update the value, then click **Update** to save the new value. Since the same library variable set may be included in multiple projects, any change to a tenant's value will also affect other projects the tenant is connected to.
+
+## Which variable templates apply to a tenant \{#which-variable-templates-apply-tenants}
+
+<TenantsWhichVariableTemplatesApply />
+
+## Library variable set templates \{#adding-a-variable-template}
+
+To specify common variables that can be used across multiple tenants, you need to add a variable template to either an existing or new library variable set:
+
+1. Navigate to **Library ➜ Variable Sets** and click **ADD VARIABLE SET**.
+2. Give the variable set a name and an optional description, and click **SAVE**.
+3. Click **Variable Templates ➜ ADD TEMPLATE**.
+4. Add the details to your template and click **SAVE**:
+
+:::figure
+![](/docs/getting-started/library-variable-sets.png "width=500")
+:::
+
+:::div{.hint}
+If you can't see any variables in the Common Variables tab, ensure you have included the library variable set in the connected project.
+:::
+
+Alternatively, you can set common variable values on an individual tenant:
+
+1. Navigate to the **Variables ➜ Common Variables** tab in the tenant screen.
+2. Provide values for each common template.
+
+:::figure
+![](/docs/projects/variables/images/variable-templates-project-template.png "width=500")
+:::
+
+## Older versions
+
+### Prior to 2024.1 \{#prior-to-2024-1}
+
+#### Project templates
```

Comment:
Do you think the older versions should just be a summary. Almost like a table with:

| Version | Change |
|---|---|
| 2024.1 | Added support for new thing |
| 2023.4 | Deprecated unsupported thing |